poppy/wfe.py

```diff
@@ -325,3 +326,65 @@ def get_opd(self, wave, units='meters'):
         if units == 'waves':
             opd /= wave.wavelength.to(u.meter).value
         return opd
+
+
+class StatisticalPSDWFE(WavefrontError):
+    """
+    Statistical PSF WFE class from power law for optical noise.
+    """
+
+    @utils.quantity_input(wfe=u.nm, radius=u.meter)
+    def __init__(self, name='PSD WFE', index=3.0, wfe=50*u.nm, radius=1*u.meter, seed=None, **kwargs):
+        """
+        Parameters
```

Comment:
This whole docstring of `__init__` should be appended/merged into the class doc string above.  Python doesn't define what should go into docstrings or where, and as a result there are some competing standards for how to do so. poppy tries to follow the numpy doc string standards, which states that the parameters to `__init__` should be documented in the class docstring. See https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

The point of this is, I suppose, that you can say `help(ClassName)` rather than `help(ClassName.__init__)`. Again this is partially a stylistic thing, but it's good to do for consistency with all the other classes in poppy.

Reply:
True, I've never seen a doctstring in the `__init__` and I have no clue why I put it there. I moved them to the class doctstring.